integration/Substra/README.md

```diff
@@ -0,0 +1,35 @@
+# Substra
+
+[Substra](https://docs.substra.org/en/latest/) is an open source federated learning framework. It provides a flexible python interface and a web app to run federated learning training at scale.
+
+Substra main usage is a production one. It has already been deployed and used by hospitals and biotech companies (see the [MELLODDY](https://www.melloddy.eu/) project for instance). Yet Substra can also be used on a single machine on a virtually splitted dataset for two use cases:
+
+- debugging code before launching experiments on a real network
+- performing FL simulations
+
+Substra was created by [Owkin](https://owkin.com/) and is now hosted by the [Linux Foundation for AI and Data](https://lfaidata.foundation/).
+
+# SubstraFL example on FLamby TCGA BRCA dataset
+
+This example illustrate the basic usage of SubstraFL, and propose a model training by Federated Learning using de Federated Average strategy on the TCGA BRCA dataset.
+
+The objective of this example is to launch a *federated learning* experiment on the six centers provided on Flamby (called organization on Substra), using the **FedAvg strategy** on the **baseline model**.
+
+This example runs in local mode, no need to deploy a Substra platform to run it.
```

Comment:
```suggestion
This example runs in local mode in python runtime, but can also be run on a deployed substra network.
```

Reply:
As flamby is not easily instalable with pip for now, the example does not support the docker mode nor the deployed mode. We might want to update the example later for both these mode?